app/views/partials/_navbar.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-inverse" role="navigation">
```

Comment:
love that the navigation bar is a partial. this makes it easy to render on every page.